src/PreCharge/GFDB.cpp

```diff
@@ -0,0 +1,117 @@
+#include <PreCharge/GFDB.hpp>
+
+
+namespace IO = EVT::core::IO;
+namespace time = EVT::core::time;
+
+namespace GFDB {
+
+    GFDB::GFDB(IO::CAN& can) : can(can){};
+
+    IO::CAN::CANStatus GFDB::init() {
```

Comment:
Ideally, these would return a new GFDBStatus type that provides a little more detail than CANStatus, but you can leave it as is for now because it works. Just be sure to make a Trello card for this to be done in the future.